MekHQ/src/mekhq/util/CamoSpec.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * MekHQ - Copyright (C) 2018 - The MekHQ Team
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ */
+package mekhq.util;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Color;
+import java.util.Optional;
+
+import megamek.client.ui.swing.util.PlayerColors;
+
+/**
+ * A simple, immutable class holding information on a camo.
+ */
+public class CamoSpec {
```

Comment:
I didn't intend to commit this file! :-(
(will remove as soon as I finish commenting)